src/test/java/me/moodcat/database/bootstrapper/BootstrapRule.java

```diff
@@ -0,0 +1,65 @@
+package me.moodcat.database.bootstrapper;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import me.moodcat.database.entities.Artist;
+import org.junit.rules.MethodRule;
+import org.junit.runners.model.FrameworkMethod;
+import org.junit.runners.model.Statement;
+
+import java.lang.reflect.Method;
+
+/**
+ * A JUnit {@link MethodRule} that ensures a certain database environment is loaded
+ * for the test.
+ *
+ * @author Jan-Willem Gmelig Meyling
+ */
+@Singleton
+public class BootstrapRule implements MethodRule {
```

Comment:
According to the [JUnit documentation](https://github.com/junit-team/junit/blob/03eff4b98a38d617a3d0cbc047221d7d337eeb86/src/main/java/org/junit/rules/MethodRule.java) the new [TestRule](https://github.com/junit-team/junit/blob/817ea24f69a040d4161525e600b73d7037a3222f/src/main/java/org/junit/rules/TestRule.java) is more suitable. Can this be swapped in for?